README.md

```diff
@@ -41,13 +41,13 @@ The following steps are required to try the "format" analyzer.
 4. Write the configuration file, e.g. `config.yml`:
 
 ```yaml
-server: 0.0.0.0:2000
+server: 0.0.0.0:9930
 db: sqlite:////tmp/lookout.sqlite
 fs: /tmp
 ```
 
 5. Run the analyzer `python3 -m lookout run lookout.style.format -c config.yml`
-6. File a fake pull request `./lookout-sdk review -v ipv4://localhost:2000`
+6. File a fake pull request `./lookout-sdk review`
```

Comment:
```suggestion
6. File a fake pull request `./lookout-sdk review ipv4://localhost:9930`
```
?

Reply:
nope. Please check the issue linked in the description.

Reply:
the all idea of the task is to remove this explicit definition of the analyzer address and keep it only for advanced users.
We also recommend for any new analyzer to use this default port (maybe not everywhere yet but we are working on it).
Our documentation already [guides a user](https://docs.sourced.tech/lookout/writing-an-analyzer/lookout-sdk) with this default in mind.

Though it's definitely up to you guys. It's only our recommendation you don't have to follow it.

Reply:
I am fine with that. Thanks for the explanation!